### PR TITLE
updating default swarm gateways

### PR DESCRIPTION
**Pull request by camronlevanger**

### CODE REVIEW

File: packages/daemon/src/vault/Vault.js [hunk 1]
```diff
@@ -172,9 +172,9 @@ export default class Vault {
       apps: new AppsRepository(),
       identities: new IdentitiesRepository(),
       settings: {
-        bzzURL: 'http://swarm-gateways.net',
-        pssURL: 'ws://localhost:8546',
-        ethURL: INFURA_URLS.ropsten,
+        bzzURL: 'http://mainframe-gateways.net:8500',
```
Comment (howleysv) [inline packages/daemon/src/vault/Vault.js]: https?

File: packages/daemon/src/vault/Vault.js [hunk 2]
```diff
@@ -172,9 +172,9 @@ export default class Vault {
...
+        pssURL: 'ws://mainframe-gateways.net:8546',
+        ethURL: INFURA_URLS.mainnet,
```
Comment (howleysv) [inline packages/daemon/src/vault/Vault.js]: @PaulLeCam What does this url apply to? Isn't there a selector in the launcher UI that the user controls? I assumed this was going all the way through the daemon

File: packages/daemon/src/vault/Vault.js (hunk 1 above)
Comment (camronlevanger) [reply packages/daemon/src/vault/Vault.js]: not available currently

**Comment by camronlevanger** [comment]
> Not sure we want to use a PSS gateway

We don't, but IIRC, something expects it to exist right now even though we don't use it.

**Pull request merged**